pkg/network/vmispec/interface.go

```diff
@@ -51,3 +51,31 @@ func lookupInterfaceByNetwork(ifaces []v1.Interface, network *v1.Network) *v1.In
 	}
 	return nil
 }
+
+// GetInterfacesNames returns slice with the names of the given interfaces.
+func GetInterfacesNames(interfaces []v1.Interface) []string {
+	ifaceNames := []string{}
```

Comment:
Without initializing `ifaceNames` when an empty `Interfaces` array is passed the function  will return nil.
I wasn't sure which way is better, also this is why on test empty arrays are passed instead of nil.

Reply:
Done